src/components/CustomerDetails/CollectCustomerDetailsScreen.test.tsx

```diff
@@ -0,0 +1,273 @@
+import {
+  render,
+  cleanup,
+  fireEvent,
+  waitFor,
+} from "@testing-library/react-native";
+import React, { FunctionComponent } from "react";
+import { Sentry } from "../../utils/errorTracking";
+import * as validateIdentification from "../../utils/validateIdentification";
+import { CampaignConfigsMap } from "../../context/campaignConfigsStore";
+import { CampaignConfigContextProvider } from "../../context/campaignConfig";
+import { AlertModalContextProvider, ERROR_MESSAGE } from "../../context/alert";
+import { CreateProvidersWrapper } from "../../test/helpers/providers";
+import { defaultFeatures, defaultProducts } from "../../test/helpers/defaults";
+import { CollectCustomerDetailsScreenContainer } from "./CollectCustomerDetailsScreen";
+import "../../common/i18n/i18nMock";
+import { IdentificationContextProvider } from "../../context/identification";
+
+jest.mock("../../utils/errorTracking");
+const mockCaptureException = jest.fn();
+(Sentry.captureException as jest.Mock).mockImplementation(mockCaptureException);
+
+const mockNavigate = { navigate: jest.fn() };
+const mockValidateAndCleanId = jest.spyOn(
+  validateIdentification,
+  "validateAndCleanId"
+);
+
+const identityDetailsInputTestId = "identity-details-input";
+const alternateIdInputTestId = "input-with-label-input";
```

Comment:
can plz explain the where the `identity-details-input` and `input-with-label-input` used in which component

Reply:
`identity-details-input` is where you input your NRIC or the main `id`. `input-with-label-input` is where you put the passport number or the `alternateId`. these variables are used in our campaign features. 
I used `alternateId` instead of passport directly because `alternateId` is not always necessary passport, however currently we only have passport as example from SSM

Reply:
solved in call